pkg/arvo/tests/lib/lv.hoon

```diff
@@ -0,0 +1,509 @@
+    %+  expect-eq
+      !>  `@lvs`0x1.4080.0000.4040.0000.4000.0000.3f80.0000.0000.0000.bf80.0000.c000.0000.c040.0000
+      !>  (make:lvs ~[.-3 .-2 .-1 .0 .1 .2 .3 .4])
+  ==
+++  test-unmake  ^-  tang
+  ;:  weld
+    %+  expect-eq
+      !>  `(list @rs)`~[.0]
+      !>  (unmake:lvs `@lvs`0x1.0000.0000)
+    %+  expect-eq
+      !>  `(list @rs)`~[.0 .0 .0]
+      !>  (unmake:lvs `@lvs`0x1.0000.0000.0000.0000.0000.0000)
+    %+  expect-eq
+      !>  `(list @rs)`(reap 5 .0)
+      !>  (unmake:lvs `@lvs`0x1.0000.0000.0000.0000.0000.0000.0000.0000.0000.0000)
+    %+  expect-eq
+      !>  `(list @rs)`(reap 16 .0)
+      !>  (unmake:lvs `@lvs`0x1.0000.0000.0000.0000.0000.0000.0000.0000.0000.0000.0000.0000.0000.0000.0000.0000.0000.0000.0000.0000.0000.0000.0000.0000.0000.0000.0000.0000.0000.0000.0000.0000)
+    %+  expect-eq
+      !>  `(list @rs)`~[.1]
+      !>  (unmake:lvs `@lvs`0x1.3f80.0000)
+    %+  expect-eq
+      !>  `(list @rs)`~[.1 .1 .1]
+      !>  (unmake:lvs `@lvs`0x1.3f80.0000.3f80.0000.3f80.0000)
+    %+  expect-eq
+      !>  `(list @rs)`(reap 5 .1)
+      !>  (unmake:lvs `@lvs`0x1.3f80.0000.3f80.0000.3f80.0000.3f80.0000.3f80.0000)
+    %+  expect-eq
+      !>  `(list @rs)`(reap 16 .1)
+      !>  (unmake:lvs `@lvs`0x1.3f80.0000.3f80.0000.3f80.0000.3f80.0000.3f80.0000.3f80.0000.3f80.0000.3f80.0000.3f80.0000.3f80.0000.3f80.0000.3f80.0000.3f80.0000.3f80.0000.3f80.0000.3f80.0000)
+    %+  expect-eq
+      !>  `(list @rs)`~[.1 .2]
+      !>  (unmake:lvs `@lvs`0x1.4000.0000.3f80.0000)
+    %+  expect-eq
+      !>  `(list @rs)`~[.1 .2 .3]
+      !>  (unmake:lvs `@lvs`0x1.4040.0000.4000.0000.3f80.0000)
+    %+  expect-eq
+      !>  `(list @rs)`~[.-3 .-2 .-1 .0 .1 .2 .3 .4]
+      !>  (unmake:lvs `@lvs`0x1.4080.0000.4040.0000.4000.0000.3f80.0000.0000.0000.bf80.0000.c000.0000.c040.0000)
+  ==
+++  test-make-unmake  ^-  tang
+  ;:  weld
+    %+  expect-eq
+      !>  `(list @rs)`~[.1 .2 .3 .4 .5]
+      !>  (unmake:lvs (make:lvs `(list @rs)`~[.1 .2 .3 .4 .5]))
+    %+  expect-eq
+      !>  `@lvs`0x1.40a0.0000.4080.0000.4040.0000.4000.0000.3f80.0000
+      !>  (make:lvs (unmake:lvs `@lvs`0x1.40a0.0000.4080.0000.4040.0000.4000.0000.3f80.0000))
+  ==
+::
+::  Tests for utility functions
+::
+++  test-length  ^-  tang
+  ;:  weld
+    %+  expect-eq
+      !>  0
+      !>  (length:lvs `@lvs`0x1)
+    %+  expect-eq
+      !>  1
+      !>  (length:lvs `@lvs`0x1.3ff0.0000)
+    %+  expect-eq
+      !>  5
+      !>  (length:lvs `@lvs`0x1.40a0.0000.4080.0000.4040.0000.4000.0000.3f80.0000)
+  ==
+++  test-make-nat  ^-  tang
+  ;:  weld
+    %+  expect-eq
+      !>  `@lvs`0x1.40a0.0000.4080.0000.4040.0000.4000.0000.3f80.0000
+      !>  (make-nat:lvs (gulf 1 5))
+  ==
+++  test-iota  ^-  tang
+  ;:  weld
+    %+  expect-eq
+      !>  `@lvs`0x1.40a0.0000.4080.0000.4040.0000.4000.0000.3f80.0000
+      !>  (iota:lvs 5)
+  ==
+++  test-isclose  ^-  tang
+  ;:  weld
+    %+  expect-eq
+      !>  (isclose:lvs .1 .1.00001 .0.0001)
+      !>  %.y
```

Comment:
As noted above, you're running into a problem here because the type of `%.y` is `%.y`, while the product of `+isclose` is `?`, which is `?(%.y %.n)`. You're hitting the nest-check in `+expect-eq`, which I earlier characterized as questionable.

You could fix this by changing using `&` instead of `%.y`. But your test sample is in the wrong order, `(expect-eq expected actual)` is the design here. If you reverse the arguments, the nest check will succeed.

There's also a bare `+expect` arm that would probably be best here, and will also avoid the argument order (and nest direction) issue.